[2023-10-17T11:19:40.489+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-17T11:19:40.499+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-17T11:19:40.500+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:40.514+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-24 00:00:00+00:00
[2023-10-17T11:19:40.523+0000] {standard_task_runner.py:57} INFO - Started process 721 to run task
[2023-10-17T11:19:40.526+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-24T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp0ox8_2pd']
[2023-10-17T11:19:40.526+0000] {standard_task_runner.py:85} INFO - Job 148: Subtask waiting_for_data
[2023-10-17T11:19:40.574+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:40.645+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-24T00:00:00+00:00'
[2023-10-17T11:19:40.656+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:40.657+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:40.657+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:40.657+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:40.668+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210524T000000, start_date=20231017T111940, end_date=20231017T111940
[2023-10-17T11:19:40.699+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:40.714+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:18.083+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-18T10:43:18.092+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-18T10:43:18.092+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:18.105+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-24 00:00:00+00:00
[2023-10-18T10:43:18.110+0000] {standard_task_runner.py:57} INFO - Started process 1692 to run task
[2023-10-18T10:43:18.113+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-24T00:00:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpka0gipxc']
[2023-10-18T10:43:18.115+0000] {standard_task_runner.py:85} INFO - Job 147: Subtask waiting_for_data
[2023-10-18T10:43:18.166+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:18.234+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-24T00:00:00+00:00'
[2023-10-18T10:43:18.247+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:18.249+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210524T000000, start_date=20231018T104318, end_date=20231018T104318
[2023-10-18T10:43:18.258+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 147 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1692)
[2023-10-18T10:43:18.285+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:18.307+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:38.610+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-18T11:43:38.641+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [queued]>
[2023-10-18T11:43:38.642+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:38.690+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-24 00:00:00+00:00
[2023-10-18T11:43:38.705+0000] {standard_task_runner.py:57} INFO - Started process 1033 to run task
[2023-10-18T11:43:38.720+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-24T00:00:00+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp_a8na564']
[2023-10-18T11:43:38.721+0000] {standard_task_runner.py:85} INFO - Job 249: Subtask waiting_for_data
[2023-10-18T11:43:38.809+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-24T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:38.961+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-24T00:00:00+00:00'
[2023-10-18T11:43:38.971+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:38.972+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:38.972+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:38.972+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:38.984+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210524T000000, start_date=20231018T114338, end_date=20231018T114338
[2023-10-18T11:43:39.042+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:39.079+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
